apps/perf-test-react-components/tsconfig.json

```diff
@@ -6,16 +6,10 @@
     "module": "commonjs",
     "jsx": "react",
     "declaration": true,
-    "sourceMap": true,
     "experimentalDecorators": true,
-    "forceConsistentCasingInFileNames": true,
-    "moduleResolution": "node",
     "preserveConstEnums": true,
-    "strictNullChecks": true,
-    "noImplicitAny": true,
-    "lib": ["es2016", "dom"],
-    "types": ["webpack-env"],
-    "skipLibCheck": true
+    "lib": ["ES2015", "DOM"],
```

Comment:
Previously this package used `es2016`, why the step back to `es2015`?

Reply:
good question, to align with `apps/perf-test`.

note: this configuration is dangerous as it emits ES5 but allows to use ES2015 features which would explode without providing polyfills which TS does not do